test/remote/gateways/remote_netbanx_test.rb

```diff
@@ -149,19 +151,21 @@ def test_successful_authorize_and_capture_with_3ds2_auth
   #   assert_equal 'OK', refund.message
   # end
 
```

Comment:
@rockyhakjoong Would it make sense to record the remote tests cassettes for the tests that are commented, and more, by delaying the requests that require the settlement to have occurred, until the settlement has occurred?
This delay could be conditional, eg to an environment variable, so recording the other cassettes in active_merchants is not impacted.

Reply:
Unfortunately, when testing a refund, you need to wait until the transaction
 if batch settled by the test system, this can take up to 2h.
 This is the reason why these tests are commented out. 

Is there some way to test these before production?

Reply:
> Is there some way to test these before production?

Not without considerable efforts.

Can your active_merchant changes not be tested in your development environment with a 2-hour delay, where required, so that the tests passes and are significant? The tests could be committed in active_merchant, skipped.

> remote tests cassettes

I made a mistake. Tests cassettes are not used in active_merchant.